widget_driver/example/lib/widgets/coffee_consumption/coffee_counter/coffee_counter_widget.dart

```diff
@@ -11,8 +13,10 @@ class CoffeeCounterWidget extends DrivableWidget<CoffeeCounterWidgetDriver> {
     return Column(
       mainAxisAlignment: MainAxisAlignment.center,
       children: [
-        Text(driver.descriptionText),
-        Text(driver.amountText),
+        Provider.value(
+          value: driver.consumedCoffeeCount,
+          child: CounterHeaderWidget(),
+        ),
```

Comment:
This is the new example which shows how the `CounterHeaderWidget` and its `CounterHeaderWidgetDriver` can depend on a buildContext value which changes.
So here the `consumedCoffeeCount` updates with each "consume button tap".

And inside the `CounterHeaderWidgetDriver` we depend on the `consumedCoffeeCount` from the buildContext using a `watch` and we get the updates correctly.